tools/LoadPackage.groovy

```diff
@@ -85,125 +85,130 @@ class LoadPackage {
+        return HttpClients.custom().setSSLSocketFactory(sslsf);
     }
 
 
-    def createAndExecuteQueryWithFileAttachment(query_push_obj_query, boundary, File object_file) {
-        def post = new org.apache.http.client.methods.HttpPost(query_push_obj_query)
+    def createAndExecuteQueryWithFileAttachment(pushObjectQuery, boundary, File objectFile) {
+        def post = new org.apache.http.client.methods.HttpPost(pushObjectQuery)
         post.addHeader("Accept", "application/json")
         post.addHeader("Content-Type", org.apache.http.entity.ContentType.MULTIPART_FORM_DATA.getMimeType() + ";boundary=" + boundary)
         post.addHeader("sessionId", this.sessionId)
 
         def builder = org.apache.http.entity.mime.MultipartEntityBuilder.create()
         builder.setBoundary(boundary);
         builder.setMode(org.apache.http.entity.mime.HttpMultipartMode.BROWSER_COMPATIBLE)
-        builder.addPart("file", new org.apache.http.entity.mime.content.FileBody(object_file))
+        builder.addPart("file", new org.apache.http.entity.mime.content.FileBody(objectFile))
         post.setEntity(builder.build())
 
         return getHttpClientBuilder().build().execute(post)
     }
 
-    def pushObject(object, package_dir, bucket_resources_list, bucket_name) {
-        def metadata_map = object.get("metadata")
-
-        // OBJECT SECTION /////////////////////////////
+    def pushObject(object, packageDir, catalogBucketResourcesList, bucketName) {
+        def packageMetadataMap = object.get("metadata")
 
         // Does the object already exist in the catalog ? -------------
-        def object_name = java.net.URLEncoder.encode(object.get("name"), "UTF-8")
-        def object_found = bucket_resources_list.find { resource -> resource.name == object_name }
-        writeToOutput(" " + object_name + " found? " + (object_found != null))
-
-        // Push the object to the bucket if it not exists
-        def object_relative_path = object.get("file")
-        def object_file = new File(package_dir.absolutePath, object_relative_path)
-        if (!object_found) {
-            // Retrieve object metadata
-            def kind = metadata_map.get("kind")
-            def commitMessageEncoded = java.net.URLEncoder.encode(metadata_map.get("commitMessage") + " (" + SCHEDULER_VERSION + ")", "UTF-8")
-            def contentType = metadata_map.get("contentType")
-            def projectName = metadata_map.get("projectName")
-            def tags = metadata_map.get("tags")
+        def objectName = java.net.URLEncoder.encode(object.get("name"), "UTF-8")
+        def objectFound = catalogBucketResourcesList.find { resource -> resource.name == objectName }
+        writeToOutput(" " + objectName + " found? " + (objectFound != null))
+
+        // Create the object file
+        def objectRelativePath = object.get("file")
+        def objectFile = new File(packageDir.absolutePath, objectRelativePath)
 
+        // Commit message
```

Comment:
explain more in comments what are you doing please

Reply:
It is a section dedicated to the commit message building